src/style/uomthesis.sty

```diff
@@ -198,7 +201,9 @@
 				{\Huge\bfseries\smallcapstitle {\@title} \par}
 					\vspace{1cm}
 				{\Large
-					{\@author} \par
+					{\@author} \par 
+						\vspace{1cm}
+					ORCID: {\@orcid} \par

```

Comment:
Would this be better as `orcid.org/{\@orcid}`?

That way a reader can go directly to the URL and view the metadata for the given ORCID, and using a lower-case URL softens the otherwise all-caps shouting of the acronym on the title page.

Going further, it might look better when reducing the `\vspace` between the name and the ORCID, and placing the ORCID in a slightly smaller font at 80% greyscale.

What do you think?